SepulcherOfTheFirstOnes/Locales/esES.lua

```diff
@@ -13,25 +13,25 @@ end
 
 L = BigWigs:NewBossLocale("Artificer Xy'mox v2", "esES")
 if L then
-	--L.traps = "Traps" -- Stasis Trap
-	--L.sparknova = "Sparknova" -- Hyperlight Sparknova
-	--L.relocation = "Tank Bomb" -- Glyph of Relocation
-	--L.relocation_count = "%s S%d (%d)" -- Tank Bomb S1 (1) // Tank Bomb (stage)(count)
-	--L.wormholes = "Wormholes" -- Interdimensional Wormholes
-	--L.wormhole = "Wormhole" -- Interdimensional Wormhole
-	--L.rings = "Rings S%d" -- Rings S1 // Forerunner Rings Stage 1/2/3/4
+	--L.traps = "Trampa de estasis" -- Stasis Trap
+	--L.sparknova = "Chispanova de hiperluz" -- Hyperlight Sparknova
+	--L.relocation = "Bomba de tank" -- Glyph of Relocation
+	--L.relocation_count = "%s S%d (%d)" -- Bomba de tank S1 (1) // Bomba de tank (stage)(count)
+	--L.wormholes = "Ruptura dimensional" -- Interdimensional Wormholes
+	--L.wormhole = "Ruptura dimensional" -- Interdimensional Wormhole
+	--L.rings = "Anillos S%d" -- Anillos S1 // Anillos genesiacos Stage 1/2/3/4
 end
 
 L = BigWigs:NewBossLocale("Dausegne, the Fallen Oracle", "esES")
 if L then
-	--L.staggering_barrage = "Barrage" -- Staggering Barrage
+	--L.staggering_barrage = "Barrera" -- Staggering Barrage
 	--L.domination_core = "Add" -- Domination Core
-	--L.obliteration_arc = "Arc" -- Obliteration Arc
+	--L.obliteration_arc = "Arco" -- Obliteration Arc
 
-	--L.disintergration_halo = "Rings" -- Disintegration Halo
-	--L.rings_x = "Rings x%d"
-	--L.rings_enrage = "Rings (Enrage)"
-	--L.ring_count = "Ring (%d/%d)"
+	--L.disintergration_halo = "Anillos" -- Disintegration Halo
+	--L.rings_x = "Anillos x%d"
+	--L.rings_enrage = "Anillos (Enrage)"
+	--L.ring_count = "Anillo (%d/%d)"
```

Comment:
This must be turned into 
`L.disintergration_halo = "Anillos" -- Disintegration Halo
L.rings_x = "Anillos x%d"
L.rings_enrage = "Anillos (Enrage)"
L.ring_count = "Anillo (%d/%d)"`
Thus uncommenting the start of the line, but leaving "--Disintegration Halo" intact